downstream/assemblies/platform/assembly-upgrading-controller.adoc

```diff
@@ -26,7 +26,7 @@ If {PlatformName} is on a version of RHEL older than RHEL 8 and you want to upgr
 
 . See _Choosing and obtaining a {PlatformNameShort} installer_ to upgrade to {PlatformNameShort} on RHEL 7.
 . Run {ControllerName} backup included in the {ControllerName} setup playbook. See link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/backup_restore.html#ag-backup-restore[Backing up and restoring {ControllerName}] in the _Automation Platform Administration Guide_ for details.
-. Obtain the {PlatformName} installer and install a fresh version of {ControllerName} on supported versions of RHEL 8.4 or higher. See link:https://access.redhat.com/support/policy/updates/errata[Red Hat Enterprise Linux Life Cycle] for information on which versions of RHEL are currently supported.
+. Obtain the {PlatformName} installer and install a fresh version of {ControllerName} on supported versions of RHEL 8.6 or higher. See link:https://access.redhat.com/support/policy/updates/errata[Red Hat Enterprise Linux Life Cycle] for information on which versions of RHEL are currently supported.
```

Comment:
I think this sentence is a bit awkward suggest something like the following:

...on supported versions of RHEL, including version 8.6 or higher.